gdx/src/com/badlogic/gdx/graphics/g2d/PolygonSpriteBatch.java

```diff
@@ -1310,7 +1310,7 @@ protected void setupMatrices () {
 		}
 	}
 
-	private void switchTexture (Texture texture) {
+	protected void switchTexture (Texture texture) {
```

Comment:
If this method is already protected in SpriteBatch, it makes sense to have the APIs be as similar as possible. Approved!